kubernetes/02-mongodb-pv.yaml

```
@@ -1,12 +1,12 @@
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: mongodb-pv
-spec:
-  capacity:
-    storage: 5Gi
-  accessModes:
-    - ReadWriteOnce
-  hostPath:
-    path: /data/mongodb
+apiVersion: v1
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)